Blog.API/Dependencies/Dependencies.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Blog.Infrastructure.Data;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Blog.Dependencies;
+
+public static class Dependencies
+{
+    public static IServiceCollection ConfigureDI(this IServiceCollection services, IConfiguration configuration)
+    {
+        services.ConfigureDatabase(configuration);
+        return services;
+    }
+
+    #region private methods
+    private static void ConfigureDatabase(this IServiceCollection services, IConfiguration configuration)
+    {
+        string connectionString = configuration.GetConnectionString("ConnectionString");
+
+        services.AddDbContext<ApiDataContext>(options => options.UseSqlServer(connectionString));
+        //services.AddDbContext<ApiDataContext>(options => options.UseMySql(connectionString, new MySqlServerVersion(new Version(8, 0 ,20))));
+    }
+    #endregion
+}
```

Comment:
Why do we need this? There is Blog.Dependencies. I think this project was meant to be deleted